### ISSUE TITLE
Doesn't work with Swift 3?

### ISSUE THREAD

**Issue by Tibb** [ThornTechPublic/InteractiveModal#6]
Thx for this great tuto, but unfortunately, it doesn't work anymore with Swift 3: the view is directly dismissed when you start to drag.
I reproduce it on my project and I also tried on with this project: same result.
Do you have any idea on how to fix that?
Thanks a lot

**Comment by nevinjethmalani** [comment]
Yes, we had the same issue. Will you be releasing a swift 3 version?

**Comment by chenr2** [comment]
Not sure when I'll be able to get to updating this tutorial. But pull requests are always welcome.

**Comment by komayip** [comment]
change 
func animationControllerForDismissedController(dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
        return DismissAnimator()
    }

to 

func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
        return DismissAnimator()
    }

works
